codespeed/mercurial.py

```diff
@@ -45,8 +45,7 @@ def getlogs(endrev, startrev):
     updaterepo(endrev.branch.project, update=False)
 
     cmd = ["hg", "log",
-            "-r", "%s:%s" % (endrev.commitid, startrev.commitid),
-            "-b", "default",
+            "-r", "%s::%s" % (startrev.commitid, endrev.commitid),
```

Comment:
The :: syntax requires the first changeset to be an ancestor of the second changeset.

    "x::y"
      A DAG range, meaning all changesets that are descendants of x and
      ancestors of y, including x and y themselves. If the first endpoint is
      left out, this is equivalent to "ancestors(y)", if the second is left
      out it is equivalent to "descendants(x)".

```"reverse(%s::%s)"``` would work.